bug-bounty-list/bug-bounty-list.json

```diff
@@ -7548,5 +7548,525 @@
+    "safe_harbor": ""
+  },
+  {
+    "program_name": "Voys",
+    "policy_url": "https://www.voys.nl/security-responsible-disclosure/#rd",
+    "submission_url": "security@wearespindle.com",
+    "launch_date": "",
+    "cash_reward": "false",
+    "swag": "false",
+    "hall_of_fame": "true",
+    "safe_harbor": "partial"
+  },
+  {
+    "program_name": "Wamba",
+    "policy_url": "http://corp.wamba.com/en/developer/security/#form",
+    "submission_url": "http://corp.wamba.com/en/developer/security/#form",
+    "launch_date": "",
+    "cash_reward": "true",
+    "swag": "false",
+    "hall_of_fame": "false",
+    "safe_harbor": ""
+  },
+  {
+    "program_name": "Webconverger",
+    "policy_url": "http://webconverger.org/security/",
+    "submission_url": "http://webconverger.org/security/",
+    "launch_date": "",
+    "cash_reward": "false",
+    "swag": "false",
+    "hall_of_fame": "true",
+    "safe_harbor": ""
+  },
+  {
+    "program_name": "Xerox",
+    "policy_url": "https://www.xerox.com/perl-bin/formeng.pl?form=product_security_information_request_7285",
+    "submission_url": "https://www.xerox.com/perl-bin/formeng.pl?form=product_security_information_request_7285",
+    "launch_date": "",
+    "cash_reward": "false",
+    "swag": "false",
+    "hall_of_fame": "false",
+    "safe_harbor": ""
+  },
+  {
+    "program_name": "Yammer",
+    "policy_url": "https://technet.microsoft.com/en-us/security/dn800983",
+    "submission_url": "https://technet.microsoft.com/en-us/security/dn800983",
+    "launch_date": "",
+    "cash_reward": "true",
+    "swag": "false",
+    "hall_of_fame": "false",
+    "safe_harbor": ""
+  },
+  {
+    "program_name": "Yandex",
+    "policy_url": "https://yandex.com/bugbounty/report/",
+    "submission_url": "https://yandex.com/bugbounty/report/",
+    "launch_date": "",
+    "cash_reward": "true",
+    "swag": "false",
+    "hall_of_fame": "true",
+    "safe_harbor": ""
+  },
+  {
+    "program_name": "Zerocopter",
+    "policy_url": "https://app.zerocopter.com/responsible_disclosure/eef4f999-2477-4802-8542-161435d30e06",
+    "submission_url": "https://app.zerocopter.com/responsible_disclosure/eef4f999-2477-4802-8542-161435d30e06",
+    "launch_date": "",
+    "cash_reward": "false",
+    "swag": "false",
+    "hall_of_fame": "false",
+    "safe_harbor": ""
+  },
+  {
+    "program_name": "Zetetic",
+    "policy_url": "https://www.zetetic.net/security/",
+    "submission_url": "support@zetetic.net",
+    "launch_date": "",
+    "cash_reward": "false",
+    "swag": "false",
+    "hall_of_fame": "false",
+    "safe_harbor": ""
+  },
+  {
+    "program_name": "Zimbra",
+    "policy_url": "https://wiki.zimbra.com/wiki/Zimbra_Responsible_Disclosure_Policy",
+    "submission_url": "https://wiki.zimbra.com/wiki/Security_Center",
+    "launch_date": "",
+    "cash_reward": "false",
+    "swag": "false",
+    "hall_of_fame": "true",
+    "safe_harbor": ""
+  },
+  {
+    "program_name": "Zynga",
```

Comment:
Isn't this a duplicate of one of the entries close to the top of your additions in this PR (`Zynga Whitehat`)? I would stick to having the program name reflect the organization's name (i.e. not including program name suffixes like "Whitehat" or "Responsible Disclosure Program" unless it's a really huge organization running multiple public programs.